packages/preview-middleware-client/test/unit/cpe/outline/nodes.test.ts

```diff
@@ -9,37 +12,33 @@ jest.mock('../../../../src/cpe/outline/utils', () => {
     };
 });
 describe('outline nodes', () => {
-    const transformNodes = (nodes: OutlineViewNode[]): Promise<OutlineNode[]> => tn(nodes);
+    const transformNodes = (nodes: OutlineViewNode[], scenario: Scenario): Promise<OutlineNode[]> =>
+        tn(nodes, scenario);
     sapCoreMock.byId.mockReturnValue({
         getMetadata: jest.fn().mockReturnValue({
-            getProperty: jest
-                .fn()
-                .mockReturnValueOnce('Component')
-                .mockReturnValueOnce('Component')
-                .mockReturnValue('')
+            getProperty: jest.fn().mockReturnValueOnce('Component').mockReturnValueOnce('Component').mockReturnValue('')
         }),
-        getProperty: jest
-            .fn()
-            .mockReturnValueOnce('Component')
-            .mockReturnValueOnce('Component')
-            .mockReturnValue('')
+        getProperty: jest.fn().mockReturnValueOnce('Component').mockReturnValueOnce('Component').mockReturnValue('')
     });
     describe('transformNodes', () => {
         test('empty tree', async () => {
-            expect(await transformNodes([])).toStrictEqual([]);
+            expect(await transformNodes([], 'UI_ADAPTATION')).toStrictEqual([]);
```

Comment:
Shouldn't this be `Adaptation_Project`?

Reply:
The test for the `Adaptation_Project` scenario is below with the name `extension point`. The one you are asking about is testing without the extension points enabled, meaning everything other than **Adaptation Project**.

Reply:
Got it, thanks!